### PR TITLE
Import updates from horus, bump PySide2 version

### PR DESCRIPTION
**Pull request by David-Estevez**
I updated grabberControls2Gui directly in [Horus](https://github.com/roboticslab-uc3m/horus) as I was in a hurry, and now I come for redemption.

### CODE REVIEW

File: programs/grabberControls2Gui/run.py [hunk 1]
```diff
@@ -23,13 +25,33 @@
 @begin.start(auto_convert=True)
 @begin.logging
 def main(remote_port: 'Remote port running the AravisGigE grabber'='/grabber'):
+    # Check for YARP network
+    yarp.Network.init()
+    if not yarp.Network.checkNetwork():
+        logging.error('Could not connect to YARP network. Please try running YARP server.')
+        quit()
```
Comment (PeterBowman) [inline programs/grabberControls2Gui/run.py:32]: Further down you use `sys.exit()`. Can we stick to only one of them?

**Review by jgvictores — APPROVED** [review]
LGTM!

File: programs/grabberControls2Gui/run.py (hunk 1 above)
Comment (jgvictores) [reply programs/grabberControls2Gui/run.py:32]: Done at https://github.com/roboticslab-uc3m/yarp-devices/commit/3c1f7f367dd637523dfe65b3335a516d5ca9d4ef !

Comment (David-Estevez) [reply programs/grabberControls2Gui/run.py:32]: Thanks!

**Pull request merged**